src/js/icons/Mail.js

```diff
@@ -4,7 +4,12 @@ import { StyledIcon } from '../StyledIcon';
 
 const Mail = forwardRef((props, ref) => (
```

Comment:
How does "Mail" fit the "social icon" theme? Less about a brand and more about the sharing mechanism?

Reply:
Good call, it does have a specific color on it when I do `plain` -- so let me confirm with Chris if it was a specific app/company.

If it's just generic, I'd opt for it not having a specified HEX color and not be considered part of this "social icon" set.

Reply:
I connected with Chris. That "Mail" option is one we have for brand to use alongside other social icons in a "share to" kind of scenario. It is not a specific company icon, so from an open-source standpoint it doesn't necessarily make sense to be grouped with this.

It seems in some of his future revisions for icons, he's exploring a "filled" version of some of the icons so that might make more sense.

Given that discussion, I'm going to remove the modification to this icon for now.